test/helpers.js

```diff
@@ -22,3 +22,111 @@ export const defaultPaginationResult = {
   total: 18,
   totalPage: 2,
 };
+
+export const paginationTest = ({ validToken, invalidToken, endpoint, model, modelMethod }) => {
```

Comment:
Separating the pagination tests into independent tests should reduce the parameters used to 3:

```suggestion
export const itReturnsTheRightPaginationValue= ({ endpoint, model, token }) => {
```